Dockerfile

```diff
@@ -2,7 +2,7 @@
 
 #Build application stage
 #We need maven.
-FROM maven:3.5.3-jdk-10
+FROM maven:3.6-jdk-12
```

Comment:
This will fix the compilation error:

```
[2020-09-01T09:42:52.002Z] [ERROR] COMPILATION ERROR : 
[2020-09-01T09:42:52.002Z] [ERROR] /usr/src/java-agent-code/apm-agent-plugins/apm-jdk-httpclient-plugin/src/main/java/co/elastic/apm/agent/httpclient/HttpRequestHeadersInstrumentation.java:[73,26] error: cannot find symbol
[2020-09-01T09:42:52.002Z]   symbol:   method of(Map<String,List<String>>,(x,y)->true)
[2020-09-01T09:42:52.002Z]   location: class HttpHeaders
[2020-09-01T09:42:52.002Z] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project apm-jdk-httpclient-plugin: Compilation failure
[2020-09-01T09:42:52.002Z] [ERROR] /usr/src/java-agent-code/apm-agent-plugins/apm-jdk-httpclient-plugin/src/main/java/co/elastic/apm/agent/httpclient/HttpRequestHeadersInstrumentation.java:[73,26] error: cannot find symbol
[2020-09-01T09:42:52.002Z] [ERROR]   symbol:   method of(Map<String,List<String>>,(x,y)->true)
[2020-09-01T09:42:52.002Z] [ERROR]   location: class HttpHeaders
[2020-09-01T09:42:52.002Z] [ERROR] -> [Help 1]
```